packages/amplication-data-service-generator/src/server/static/src/auth/auth.module.ts

```diff
@@ -8,10 +9,31 @@ import { AuthResolver } from "./auth.resolver";
 import { AuthService } from "./auth.service";
 import { BasicStrategy } from "./basic.strategy";
 import { PasswordService } from "./password.service";
+import { JwtStrategy } from "./jwt/jwt.strategy";
+import { SecretsManagerModule } from "../providers/secrets/secretsManager.module";
+import { SecretsManagerService } from "../providers/secrets/secretsManager.service";
 
+export const JWT_SECRET = "JWT_SECRET";
 @Module({
-  imports: [forwardRef(() => UserModule), PassportModule],
-  providers: [AuthService, BasicStrategy, PasswordService, AuthResolver],
+  imports: [
+    forwardRef(() => UserModule),
+    PassportModule,
+    SecretsManagerModule,
+    JwtModule.registerAsync({
+      imports: [SecretsManagerModule],
+      useFactory: async (secretsService: SecretsManagerService) => ({
+        secret: secretsService.getSecret<string>(JWT_SECRET),
+        signOptions: { expiresIn: "2d" },
```

Comment:
Read from config

Reply:
i didn't found the config file, or you what to give the user to set in as an app settings prop